catalogue_graph/src/adapters/ebsco/transformers/ebsco_to_weco.py

```diff
@@ -1,5 +1,6 @@
 from datetime import datetime
```

Comment:
Include for suggestion on l35
```suggestion
from datetime import datetime, UTC
```